exchanges/bitmex/bitmex_websocket.go

```diff
@@ -65,9 +66,18 @@ const (
 	bitmexActionUpdateData  = "update"
 )
 
-var subscriptionNames = map[string]string{
-	subscription.OrderbookChannel: bitmexWSOrderbookL2,
-	subscription.AllTradesChannel: bitmexWSTrade,
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Channel: bitmexWSAnnouncement},
```

Comment:
```
[DEBUG] | WEBSOCKET | 06/09/2024 09:12:01 | Bitmex websocket connection: sending message: {"op":"subscribe","args":["announcement","orderBookL2:UNI_USDT","orderBookL2:ETHUSD","orderBookL2:XBTH25","trade:UNI_USDT","trade:ETHUSD","trade:XBTH25"]}
[DEBUG] | WEBSOCKET | 06/09/2024 09:12:01 | Bitmex websocket connection: message received: {"status":400,"error":"Topic announcement is not available on this endpoint. Please see the documentation at https://www.bitmex.com/app/wsAPI.","meta":{},"request":{"op":"subscribe","args":["announcement","orderBookL2:UNI_USDT","orderBookL2:ETHUSD","orderBookL2:XBTH25","trade:UNI_USDT","trade:ETHUSD","trade:XBTH25"]}}
```
It highlights that we do not recognise WS errors properly since it comes through as `DEBUG`

Reply:
```
[DEBUG] | WEBSOCKET | 06/09/2024 09:16:45 | Bitmex websocket connection: message received: {"status":400,"error":"Unknown table: privateNotifications. Please see the documentation at https://www.bitmex.com/app/wsAPI.","meta":{},"request":{"op":"subscribe","args":["affiliate","order","margin","privateNotifications","transact","wallet","execution:ETHUSD","position:ETHUSD"]}}
```

https://www.bitmex.com/app/wsAPI

> The following subscriptions are only available on the /realtimePlatform endpoint:
> "announcement",        // Site announcements
> "privateNotifications", // Individual notifications - currently not used (requires auth)

We are not connecting to the additional endpoint for those subscriptions. I am not asking you to set that up, but I don't think those two topics should be part of this PR

Since these messages came through as `DEBUG` I get how they were missed

Reply:
Fixed 0db29723

* Announcement and Private are no longer included
* Switched to using the multiplexer to make it safe to get auth and stream failures out with separately without racing
* Error handling, including auth

Had to rebase onto master to get `SendMessageReturnResponses` unfortunately.